packages/kyt-runtime/client.js

```diff
@@ -1,3 +1,3 @@
-const { preloadReady } = require('./lib/loadable');
+var { preloadReady } = require('./lib/loadable');
```

Comment:
I don't think the destructuring is going to get digested either.
I'll open another PR to fix it.

Reply:
Yeah IE is throwing another error: expected identifier.